commander/tracker/tracking.go

```diff
@@ -52,6 +53,8 @@ func (t *Tracker) startCheckingTxs(ctx context.Context) error {
 			return nil
 		default:
 			if t.isEmptyTxsQueue() {
+				dur, _ := time.ParseDuration("100ms")
+				time.Sleep(dur)
```

Comment:
this change is unrelated but adding it to this PR to prevent a proliferation of PRs.

without this change this goroutine spinloops, pegging a cpu at 100%. with this change we're still doing unnecessary work but we're only doing it every 100ms. Not good enough for a mobile app but more than fine for a server

Reply:
I think the `ParseDuration` is unnecessary here, will remove it